.github/workflows/publish.yml

```diff
@@ -0,0 +1,27 @@
+name: Publish to NPM
+on:
+  release:
+    types: [published]
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - name: 'Check out the repo'
+        uses: actions/checkout@v3
```

Comment:
Instead of using tag can we use specific commit hash for all the uses ?

Reply:
Sure, I believe we can use this hash https://github.com/BirthdayResearch/quantum/blob/main/.github/workflows/ci.yml#L33